extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -134,3 +134,16 @@ export function doubleclickIsA4AEnabled(win, element) {
   }
   return enableA4A;
 }
+
+export const adConfig = {
```

Comment:
Also, where are you seeing clientIdScope being used? When I search for it I don't get anything back related to FF.